docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -62,10 +62,8 @@ web3.eth.accounts.wallet.add(account);
 In order to transfer ownership of your Universal Profile, you need to initialize your current LSP6 Key Manager.
 
 ```js
-const oldKeyManager = new web3.eth.Contract(
-  LSP6KeyManager.abi,
-  keyManagerAddress,
-);
+//prettier-ignore
```

Comment:
i think if you put it here it will show in the page. This is what i meant:

https://github.com/lukso-network/docs/blob/2b56b521020f1a48468599bfbf4874545aad8d62/docs/guides/browser-extension/sign-in-with-ethereum.md?plain=1#L63